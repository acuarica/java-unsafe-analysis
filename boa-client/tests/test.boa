
p: Project = input;

countsTotal: output sum of int;
countsJava: output sum of int;

countsTotal << 1;

foreach (i: int; match(`^java$`, lowercase(p.programming_languages[i]))) {
	countsJava << 1;
}
