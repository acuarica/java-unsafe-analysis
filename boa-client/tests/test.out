countsJava[] = 493
countsTotal[] = 7029
